hack/update-generated-runtime-dockerized.sh

```diff
@@ -40,22 +40,22 @@ if [[ -z "$(which protoc)" || "$(protoc --version)" != "libprotoc 3."* ]]; then
 fi
 
 function cleanup {
-	rm -f ${KUBE_REMOTE_RUNTIME_ROOT}/api.pb.go.bak
+	rm -f "${KUBE_REMOTE_RUNTIME_ROOT}/api.pb.go.bak"
 }
 
 trap cleanup EXIT
 
-gogopath=$(dirname $(kube::util::find-binary "protoc-gen-gogo"))
+gogopath=$(dirname "$(kube::util::find-binary "protoc-gen-gogo")")
 
 PATH="${gogopath}:${PATH}" \
   protoc \
   --proto_path="${KUBE_REMOTE_RUNTIME_ROOT}" \
   --proto_path="${KUBE_ROOT}/vendor" \
-  --gogo_out=plugins=grpc:${KUBE_REMOTE_RUNTIME_ROOT} ${KUBE_REMOTE_RUNTIME_ROOT}/api.proto
+  --gogo_out=plugins=grpc:"${KUBE_REMOTE_RUNTIME_ROOT}" "${KUBE_REMOTE_RUNTIME_ROOT}/api.proto"
 
 # Update boilerplate for the generated file.
-echo "$(cat hack/boilerplate/boilerplate.generatego.txt ${KUBE_REMOTE_RUNTIME_ROOT}/api.pb.go)" > ${KUBE_REMOTE_RUNTIME_ROOT}/api.pb.go
+cat hack/boilerplate/boilerplate.generatego.txt "${KUBE_REMOTE_RUNTIME_ROOT}/api.pb.go" > tmpfile && mv tmpfile "${KUBE_REMOTE_RUNTIME_ROOT}/api.pb.go"
```

Comment:
change reason:
```
In hack/update-generated-runtime-dockerized.sh line 57:
echo "$(cat hack/boilerplate/boilerplate.generatego.txt ${KUBE_REMOTE_RUNTIME_ROOT}/api.pb.go)" > ${KUBE_REMOTE_RUNTIME_ROOT}/api.pb.go
     ^-- SC2005: Useless echo? Instead of 'echo $(cmd)', just use 'cmd'.
                                                        ^-- SC2086: Double quote to prevent globbing and word splitting.
                                                                                                  ^-- SC2086: Double quote to prevent globbing and word splitting.
```

```
In hack/update-generated-runtime-dockerized.sh line 57:
cat hack/boilerplate/boilerplate.generatego.txt "${KUBE_REMOTE_RUNTIME_ROOT}/api.pb.go" > "${KUBE_REMOTE_RUNTIME_ROOT}/api.pb.go"
                                                ^-- SC2094: Make sure not to read and write the same file in the same pipeline.
                                                                                          ^-- SC2094: Make sure not to read and write the same file in the same pipeline.
```